src/main/resources/i18n/messages.properties

```diff
@@ -248,6 +248,26 @@ EditUmsConfigurationFileManually=Edit UMS configuration file manually
 Empty=Empty
 Enable=Enable
 EnableAvisynthVariableFramerate=Enable AviSynth variable framerate change into a constant framerate (convertfps=true)
+2Dto3DConversionSettings=2D to 3D Conversion Settings
```

Comment:
The keys look good now, can you please sort it alphabetically?

Reply:
Done. I have sorted all of the keys in the file and run UMS to test.